homeassistant/components/cloud/prefs.py

```diff
@@ -374,6 +378,12 @@ def cloud_ice_servers_enabled(self) -> bool:
         )
         return cloud_ice_servers_enabled
 
+    @property
+    def backup_sync_enabled(self) -> bool:
+        """Return if backup sync is enabled."""
+        backup_sync_enabled: bool = self._prefs.get(PREF_ENABLE_BACKUP_SYNC, False)
```

Comment:
This means existing users will not have backup sync enabled, I'm not sure if that's correct or not.